apps/ui/src/apis/communities/fetch-community-user-status.ts

```diff
@@ -0,0 +1,73 @@
+import { UserStatus } from '@communetypes/Enums';
+import { UserCommunitiesStatus } from '../../types/community-type';
+import { graphqlRequest } from '../graphql';
+
+interface UserCommunityNode {
+  communityId: number;
+  status: string;
+  userId: number;
+}
+interface communityOwnerNode {
+  id: number;
+  ownerId: number;
+  title: string;
+}
+
+interface AllUserCommunitiesResponse {
+  allUserCommunities: {
+    nodes: UserCommunityNode[];
+  };
+  allCommunities: {
+    nodes: communityOwnerNode[];
+  };
+}
+
+const GET_USER_COMMUNITIES_STATUS = `
+query GetUserCommunitiesStatus($userId: Int!) {
+  allUserCommunities(condition: {userId: $userId}) {
+    nodes {
+      communityId
+      status
+      userId
+    }
+  }
+   allCommunities(condition: { ownerId: $userId }) {
+    nodes {
+      id
+      ownerId
+      title
+    }
+  }
+}
+`;
+
+const managerStatus = 'Manager' as UserStatus;
```

Comment:
use the enum here

Reply:
It does something weird... if I do it like this the app crashes and the error is that something failed importing UserStatus